src/styles/common/_commonIndex.scss

```diff
@@ -0,0 +1,7 @@
+@import "variables";
+@import "navbar";
+@import "footer";
+@import "login";
+@import "landing";
```

Comment:
Landing is not a common style, in fact, it is the styles of a specific component calling LandingSection. A better practice would be to colocate the styles for landing along with the landingSection component only, e.g. in `src/components/LandingSection/LandingSection.scss`. In fact, sass modules can be used in this situation

Reply:
I actually had a few doubts in how styles are to be defined, I looked at some existing repos and there were different ways the styles were added. Some had all their styles in the styles folder itself and used _commonIndex to export the styles. Like this repo here : https://github.com/Bahmni/implementer-interface/tree/master/styles. While others did not have a separate styles folder and the styles for each one were defined along with the js file itself, like so : https://github.com/Bahmni/bahmni-lab-frontend/tree/main/. So which styling convention would be appropriate for this react app?

Reply:
The modern way is to colocate the styles with the component that is using it. Many of these repositories are quite old so use older techniques. Better to colocate styles so that it is easier to locate/refactor/delete. In fact, look up css-modules. we have than enabled for this repository, that will give you isolated css and better separation